lib/combo.coffee

```diff
@@ -667,6 +663,21 @@
     updateClassNames: () ->
       if @classNameOnEmpty
         @input.toggleClass @classNameOnEmpty, not @getRawValue()
+                
+    getLabel: (item) ->
+      label = @label?(item,  @getRawValue())
+      if label? then "<span class='#{label.className}'>#{label.text}</span>" else ""
+    
+    toggleInputLabel: () ->
+      if @isExpanded or @getRawValue() == ""

```

Comment:
Det ville være pænere hvis du havde tre conditionals

if (@isExpanded or @getRawValue() == "") and not @inputLabel? return